<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Step</class>
 <widget class="QWidget" name="Step">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>881</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string> 工步参数 </string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_50">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_S_Angle">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>30</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_S_Yaxis">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>110</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_S_Xaxis">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>150</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_S_AngleCompensate">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>70</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_S_XaxisCorrect">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>190</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_boardThickness_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>角度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mould_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y轴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mould_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X轴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_material_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>角度补偿</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mould_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X轴校准</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_S_distance">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>30</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_S_return">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>110</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_S_Holding">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>150</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_S_pressure">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>70</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_S_Raxis">
    <property name="geometry">
     <rect>
      <x>575</x>
      <y>190</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_boardThickness_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退让距离</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mould_8">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>回程时间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mould_7">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保压时间</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_material_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>压力</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_mould_9">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>200</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>R轴</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_51">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>160</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_52">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>200</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_53">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>40</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_54">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>80</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>KN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>120</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_56">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>160</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>S</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_57">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>200</y>
      <width>58</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_58">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_59">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>°</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableWidget_Step">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>881</width>
     <height>261</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="autoScrollMargin">
    <number>16</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>11</number>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>27</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="verticalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <row/>
   <column>
    <property name="text">
     <string>工步号</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>角度</string>
    </property>
    <property name="font">
     <font>
      <family>文泉驿正黑</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
    <property name="icon">
     <iconset resource="A66-appsrc.qrc">
      <normaloff>:/ICO/角度.ico</normaloff>:/ICO/角度.ico</iconset>
    </property>
   </column>
   <column>
    <property name="text">
     <string>补偿</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
    <property name="icon">
     <iconset resource="A66-appsrc.qrc">
      <normaloff>:/ICO/角度补偿.ico</normaloff>:/ICO/角度补偿.ico</iconset>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Y轴</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>X轴</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>X轴校准</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>退让距离</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>压力</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>回程时间</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>保压时间</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>R轴</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignLeft|AlignVCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QToolButton" name="pushButton_Left_7">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>408</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_Left_5">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>282</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_Left_6">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>345</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_Left_1">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>UP</string>
   </property>
   <property name="icon">
    <iconset resource="A66-appsrc.qrc">
     <normaloff>:/ico/ICO/P2-UP.png</normaloff>:/ico/ICO/P2-UP.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_Left_8">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>471</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_Left_3">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>144</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="icon">
    <iconset resource="A66-appsrc.qrc">
     <normaloff>:/ico/ICO/P1-DEL.png</normaloff>:/ico/ICO/P1-DEL.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_Left_2">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>77</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>新增</string>
   </property>
   <property name="icon">
    <iconset resource="A66-appsrc.qrc">
     <normaloff>:/ico/ICO/P1-NEW.png</normaloff>:/ico/ICO/P1-NEW.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="pushButton_Left_4">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>210</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>down</string>
   </property>
   <property name="icon">
    <iconset resource="A66-appsrc.qrc">
     <normaloff>:/ico/ICO/P2-DOWN.png</normaloff>:/ico/ICO/P2-DOWN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>36</width>
     <height>36</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="A66-appsrc.qrc"/>
 </resources>
 <connections/>
</ui>
